scripts/cli2md.rb

```diff
@@ -2,37 +2,51 @@
 
 require "cgi"
 
-incode = false
-first_param = false
+# are we in a code block?
+in_code = false
+
+# are with in the table of parameters?
+in_table = false
 
 # read from stdin or files in the args
 ARGF.each_with_index do |line, line_num|
-  # Headings
+  # Replace all prime symbols with backticks. We use prime symbols instead of backticks in CLI
+  # helptext because Go does not support escaping backticks in backtick delimited strings.
```

Comment:
> because Go does not support escaping backticks in backtick delimited strings

this is true, but not why we avoid backticks in usage strings; we actually avoid them because urfave/cli has [special handling](https://cli.urfave.org/v1/examples/flags/#placeholder-values) for backtick-enclosed substrings in usage docs, and mangles usage strings that have backticked strings in them

Reply:
admittedly i probably could have put that there instead of just writing "reasons"